frontend/src/views/RegisterPage.vue

```diff
@@ -40,7 +40,13 @@
                 <p style="color: red; font-size: 12px;">{{errMessage}}</p>
             </div>
             <div class="terms">
-              <div><input v-model="isTermsAgreed" type="checkbox"> <span>I agree to all terms</span></div>
+              <div><input v-model="isTeacher" type="checkbox"><span>I am a teacher</span></div>
```

Comment:
Is it worth having this text say "I am a teacher/lecturer" or just "I am a lecturer" since the app is aimed at Universities?

Reply:
Okay